### ISSUE TITLE
Disable wrapping if elses

### ISSUE THREAD

**Issue by OscarGodson** [prettier/prettier#3537]
**Prettier 1.9.2**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEBLAZgAgBQAcCGATgM5wCSs2AbvgDYCucAlJgLzuYCMmAPj5gRLlKNBszYcADC2AAdKJkyE4MeoQUBCDaMYBueQF95cWqUwYcOuBNaYA5DEKM7vflZv30dUnZnzFyqrqmO7stg5OcHb6UEZQJmYW2DAAnrhwEFihHHYARhAQtHD4UL6YcgpKKmoKVjFxCdYVAdXB9FAAJnDoqPEd9SAANCAQuDCo0MTIoLiEvTAA6qgdMAAWyAAcksMw+LlLK+tIAEzDxL0A5kUAivQQ8MhepnA7hPiotJcAwhAAtr-4ZAgKDQOBDEC5N5gADWKgAygQwJdkI5GMMAFbEAAeACEobCYHD8L84AAZXpgpBPUjDQSkQhA3J7FK0aDg0i-VAoyLDeikAAqeymVO8LzQUHpMAACm8LgDHqLhsoAI70VDKGX4OWAkXPWmECCkBZvXBA2ZwelUMHDIgGgDuUqICGFIHwVAgy3BHQgYAVetdxBgfppI3oMFwYeOwbgBgMQA)

**Input:**
```js
if (parseInt(value) === 1 || parseInt(value) === 0) {
  return !!value;
}
else if (value === 'true' || value === 'false') {
  return value === 'true';
}
else if (typeof value === 'boolean') {
  return value;
}
else {
  return undefined;
}
```

**Output:**
```js
if (parseInt(value) === 1 || parseInt(value) === 0) {
  return !!value;
} else if (value === "true" || value === "false") {
  return value === "true";
} else if (typeof value === "boolean") {
  return value;
} else {
  return undefined;
}

```

**Expected behavior:**

I'd like to disable or stop it from wrapping those else if's. The code above is for some formatting function (you give it something and it tries hard to force a boolean) and to me that's incredibly hard to parse visually. It all just blends together. And if I add comments it puts them inside the `{}` which still doesnt make it easier to scan. Is there a way to disable that so it looks like the above with new lines?

<img width="142" alt="prettier 2017-12-19 16-33-55" src="https://user-images.githubusercontent.com/20615/34185506-93953346-e4da-11e7-82e2-4a05be381c42.png">

**Comment by lydell** [comment]
Hi! Thanks for the issue! This seems to be a duplicate of #840, so I'm closing.
